draft-ietf-mls-protocol.md

```diff
@@ -818,35 +818,21 @@ strings with length Hash.length to HPKE key pairs.
 Ciphersuites are represented with the CipherSuite type. HPKE public keys
 are opaque values in a format defined by the underlying Diffie-Hellman
 protocol (see the Ciphersuites section of the HPKE specification for more
-information):
+information).
 
 ~~~~~
-enum {
-    P256_SHA256_AES128GCM(0x0000),
-    X25519_SHA256_AES128GCM(0x0001),
-    (0xFFFF)
-} CipherSuite;
```

Comment:
Now the CipherSuite type is never defined

Reply:
It is defined in section "MLS Ciphersuites" as a two byte value now.